addons/expanding-search-bar/addon.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Expanding search bar",
+  "description": "Expands the website search bar when you click on it.",
```

Comment:
Maybe change this to `Expands the website search bar when your focused on it.`?

Reply:
wouldn't that need hover then?